project/.bloop/terraapi-build.json

```diff
@@ -0,0 +1,1509 @@
+{
+    "version" : "1.1.0-M2",
+    "project" : {
+        "name" : "terraapi-build",
+        "directory" : "/home/shane/code/TerraApi/project",
+        "sources" : [
+            "/home/shane/code/TerraApi/project/src/main/scala-2.12",
```

Comment:
Shouldn't this be using relative filepaths? I'm not going to have `/home/shane/code/...`

Reply:
I need to remove this. I thought the bloop files were gitignored.

Reply:
Its just junk